lib/journaled/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Journaled
-  VERSION = "5.3.1".freeze
+  VERSION = "5.3.2"
```

Comment:
I'm only bumping patch because our actual dependencies were already sitting on 7.1 versions and I'm not dropping support for anything. We simply didn't actually test in CI.